loader/CMakeLists.txt

```diff
@@ -29,8 +34,28 @@ foreach(test ${test_targets})
   target_compile_definitions(${test} PRIVATE
     "IGNDummyPlugins_LIB=\"$<TARGET_FILE:IGNDummyPlugins>\"")
 
+  target_compile_definitions(${test} PRIVATE
+    "IGN_PATH=\"${IGNITION-TOOLS_BINARY_DIRS}\"")
+
+  target_compile_definitions(${test} PRIVATE
+    "IGN_VERSION_FULL=\"${PROJECT_VERSION_FULL}\"")
+
 endforeach()
 
+if(TARGET UNIT_ign_TEST)
+  set(_env_vars)
+  list(APPEND _env_vars "IGN_CONFIG_PATH=${CMAKE_BINARY_DIR}/test/conf")
+
+  set_tests_properties(UNIT_ign_TEST PROPERTIES
+    ENVIRONMENT "${_env_vars}")
+endif()
+
 install(
   DIRECTORY include/
   DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
+
+#============================================================================
+# ign command line support
+#============================================================================
+add_subdirectory(conf)

```

Comment:
The dependency is not `REQUIRED`, and I saw that you checked for its presence to add the test. Should it be checked when adding these directories too?

Reply:
this folder only installs configuration files, which doesn't require `ignition-tools` to be installed, so I think it's ok to always install them

Reply:
I changed the `src/cmd` folder to always be included because it is also only configuring files and changed the test to be disabled on windows